src/component/handlers/edit/editOnInput.js

```diff
@@ -88,8 +89,41 @@ function editOnInput(editor: DraftEditor): void {
   var domSelection = global.getSelection();
 
   var {anchorNode} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
-    return;
+  const isNotTextNode =
```

Comment:
To that end, our version of `DraftFeatureFlags` is the same as Facebook's. This is the code running on Facebook itself.